packages/pages/src/dev/server/ssr/yextEnv.ts

```diff
@@ -28,34 +28,23 @@ export function getEnvFromYextCredential(): YextEnv {
       return "production";
     }
 
-    if (
-      typeof data === "object" &&
-      data !== null &&
-      "env" in data &&
-      typeof (data as any).env === "string"
-    ) {
-      const env = (data as { env: string }).env as string;
-
-      const validEnvs: YextEnv[] = [
-        "development",
-        "qa",
-        "sandbox",
-        "production",
-      ];
-      if (validEnvs.includes(env as YextEnv)) {
+    const env = (data as { env: string }).env as string;
```

Comment:
Again, just do `const env = data['env']`. It's valid json if we got here. Your type casting here doesn't provide value because you just do `as string` anyways, so regardless it will be treated as a string no matter what the object were to be.